.gitignore

```diff
@@ -11,6 +11,7 @@ traffic_ops/app/public/CRConfig-Snapshots/*
 traffic_ops/app/public/Trafficserver-Snapshots/*
 docs/.todo_cache
 docs/_build/
+docs/source/_build/
```

Comment:
Why was this necessary?

Reply:
I used a VSCode extension for reStructuredText (https://marketplace.visualstudio.com/items?itemName=lextudio.restructuredtext) while working on this PR. It built the docs in that directory for its live preview feature.

Reply:
Well then personally I think you should take this out, because it's not a part of our build system.

Reply:
I disagree - we shouldnt drive what development processes developers use. There is a lot of IDEs that put files/directories in place in the repo and I rather have extra stuff in the .gitignore (which does no harm) then accidentally push something unwanted.

FWIW I use the same extension

Reply:
Personally, I'm not opposed to adding common dev tool files to .gitignore to make people's lives easier.

It adds a tiny bit of safety too, so people don't have to delete them to have 0 "untracked files."

Reply:
Well it's one thing to support specific IDEs and/or text editors - and we do have things like` .vscode/` in that `.gitignore`, but idk about supporting all the plugins people like to use for each. We already have a `docs/source/_ext` directory that's actually used, so it's possible we could want something for a similar purpose dealing with the build to be named that.

But all that said, I won't block the PR on this.